beacon-chain/sync/initial-sync/service.go

```diff
@@ -237,14 +239,14 @@ func (s *Service) Resync() error {
 
 	// Set it to false since we are syncing again.
 	s.synced.UnSet()
-	defer func() { s.synced.Set() }()                       // Reset it at the end of the method.
-	genesis := time.Unix(int64(headState.GenesisTime()), 0) // lint:ignore uintcast -- Genesis time will not exceed int64 in your lifetime.
+	defer func() { s.synced.Set() }()                            // Reset it at the end of the method.
+	s.genesisTime = time.Unix(int64(headState.GenesisTime()), 0) // lint:ignore uintcast -- Genesis time will not exceed int64 in your lifetime.
```

Comment:
do we need to reset this ? Looks like we only needed it before because we are passing it in as an argument

Reply:
You're right, this line can just be removed.